### PR TITLE
[PT FE] Support aten::softplus

### PR DESCRIPTION
**Pull request by mvafin**
### Details:
 - *Support aten::softplus*

### Tickets:
 - *None*

### CODE REVIEW

**Review by eaidova — APPROVED** [review]

File: src/frontends/pytorch/src/op_table.cpp [hunk 1]
```diff
@@ -498,6 +498,7 @@ const std::map<std::string, CreatorFunction> get_supported_ops_ts() {
         {"aten::size", op::translate_size},
         {"aten::slice", op::quantizable_op<op::translate_slice>},
         {"aten::softmax", op::translate_softmax},
+        {"aten::softplus", op::translate_1to1_match_1_inputs<opset10::SoftPlus>},
```
Comment (rkazants) [inline src/frontends/pytorch/src/op_table.cpp:501]: can we avoid use of `opset` namespace and specify concrete version of `Softplus`?

File: tests/layer_tests/pytorch_tests/test_softplus.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+class aten_softplus(torch.nn.Module):
+    def forward(self, x):
+        return torch.nn.functional.softplus(x)
+
+
+class TestSoftplus(PytorchLayerTest):
+    def _prepare_input(self):
+        import numpy as np
+        return (np.random.randn(2, 4, 224, 224).astype(np.float32),)
```
Comment (rkazants) [inline tests/layer_tests/pytorch_tests/test_softplus.py:18]: only `float32` is supported?

Comment (mvafin) [reply tests/layer_tests/pytorch_tests/test_softplus.py:18]: Yes, torch operation requires fp32

File: src/frontends/pytorch/src/op_table.cpp (hunk 1 above)
Comment (mvafin) [reply src/frontends/pytorch/src/op_table.cpp:501]: This file have many imports, agree it makes sense to import specific operation, but that is not a scope of this PR.

**Pull request merged**